jvector-examples/src/main/java/io/github/jbellis/jvector/example/Grid.java

```diff
@@ -265,7 +270,6 @@ private static ResultSummary performQueries(ConfiguredSystem cs, int topK, int e
                 var searcher = cs.getSearcher();
                 var sf = cs.scoreProviderFor(queryVector, searcher.getView());
                 sr = searcher.search(sf, efSearch, Bits.ALL);
-
                 var gt = cs.ds.groundTruth.get(i);
```

Comment:
the newline separated "run the search" from "process the results" but explicit comments would be better